README.md

```diff
@@ -302,6 +302,19 @@ Passing in 0 disables exponential backoff completely. In that case, the client w
 
 
 ## Upgrading
+
+### v93 -> v94
+- An event's chart key can no longer be changed
+
+### v92 -> v93
+- Objects can be released in batch
```

Comment:
I would rephrase this to:

"Removed `StatusChangeRequest` constructor. Use setters instead."